src/App.tsx

```diff
@@ -1,61 +1,139 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+import { TodoList } from './components/TodoList';
+
+function findUser(id: number): User | null {
+  return usersFromServer.find(user => user.id === id) || null;
+}
+
+const todoList: Todo[] = todosFromServer.map(todo => (
+  {
+    ...todo,
+    user: findUser(todo.id),
+  }
+));
+
+function calculateId(todos: Todo[]): number {
```

Comment:
Consider renaming the `calculateId` function to `generateNextId`/`getNextId` for better clarity on its purpose.